appveyor.yml

```diff
@@ -1,65 +1,86 @@
 version: '{branch}-{build}'
+build: off
 
 platform:
   - x86
   - x64
 
 environment:
   global:
-    PYTHONPATH: "C:\\testdir"
-    PYTHONWARNINGS: "ignore:::wheel.pep425tags:"
-    CONDA_BLD_VERSION: "3.5"
-    CONDA_BLD: "C:\\conda"
-    NUNIT: "nunit-console"
+    PYTHONUNBUFFERED: True
+    PYTHONWARNINGS: 'ignore:::wheel.pep425tags:'
+    PYTHONPATH: C:\testdir
+    NUNIT: nunit-console
+    CONDA_BLD: C:\conda
+    CONDA_BLD_VERSION: 3.5
 
     # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
     # /E:ON and /V:ON options are not enabled in the batch script interpreter
     # See: http://stackoverflow.com/a/13751649/163740
-    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci\\run_with_env.cmd"
+    CMD_IN_ENV: 'cmd /E:ON /V:ON /C .\ci\run_with_env.cmd'
 
   matrix:
-    - PYTHON_VERSION: "2.7"
-    - PYTHON_VERSION: "3.3"
-    - PYTHON_VERSION: "3.4"
-    - PYTHON_VERSION: "3.5"
-    - PYTHON_VERSION: "3.6"
+    - PYTHON_VERSION: 2.7
+    - PYTHON_VERSION: 3.3
+    - PYTHON_VERSION: 3.4
+    - PYTHON_VERSION: 3.5
+    - PYTHON_VERSION: 3.6
```

Comment:
They are still being used as strings. Remeber this is doing set PYTHON_VERSION=3.3 which comes out as a string when you do `%PYTHON_VERSION%`